### DISCUSSION TITLE
Kobo Clara Colour Screen

### DISCUSSION THREAD

**Discussion by delbronski** [vroland/epdiy#336] [Ideas]
Hello, 

I was walking through a shop the other day and came across Kobo's new colour e-readers (Kobo Clara Colour). 

You can order a replacement screen from [ifixit](https://eustore.ifixit.com/products/kobo-clara-colour-screen-assembly-genuine?variant=48041498837332)

Right now it is impossible to order a colour e-ink display that does not take 20+ seconds to refresh. It would be pretty cool if the epdiy could work with a cheap colour screen like this. How feasible do you think that this?

**Comment by martinberlin** [comment]
It could work but first it would be essential to have the datasheet of the display and know what connector it uses.

> You can order a replacement screen from ifixit

Also notice that it says **Out of Stock** so I don't think you can order a replacement unless they have some stock

**Reply by delbronski** [reply]
Ah yeah you are right. Didn't notice the out of stock text. I dug a bit and it seems to be the[ Kaleido 3 Plus screen ](https://www.eink.com/brand/detail/Kaleido3)

I could not find a datasheet online for the colour version.
